plugins/plugin-java-debugger/che-plugin-java-debugger-server/src/main/java/org/eclipse/che/plugin/jdb/server/JavaDebugger.java

```diff
@@ -792,4 +810,30 @@ private ThreadStatus toThreadStatus(int status) {
         return ThreadStatus.UNKNOWN;
     }
   }
+
+  private SuspendPolicy toSuspendPolicy(int suspendEventRequest) {
```

Comment:
Consider enum constructor please.

Reply:
Not the case. Here we convert java debugger constant into model.
Other implementations might have different constants.

Reply:
I see...